build/export-all-patch.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+VERSION=$(cat ~/bromite/build/RELEASE)
+CURRENT_RELEASE=$(git -C ~/chromium/src/ rev-parse --verify refs/tags/$VERSION)
+
+ALLPATCHS=$(git -C ~/chromium/src/ rev-list HEAD...$CURRENT_RELEASE)
+
+for patch in $ALLPATCHS; do
+
+	PATCH_FILE=$(git -C ~/chromium/src/ show -s $patch | grep FILE: | sed 's/FILE://g' | sed 's/^[ \t]*//;s/[ \t]*$//')
```

Comment:
@csagan5 
here I take the file name of the patch inserted above and trim spaces